apps/gallery/locales/gallery.en-US.properties

```diff
@@ -91,5 +91,6 @@ preview = Preview
 # Header format used in thumbnail view
 # see http://pubs.opengroup.org/onlinepubs/007908799/xsh/strftime.html
 date-group-header = %B %Y
-# Date time format for each individual photo's description
-image-created-at = Image created at %I:%M %p, %B %e, %Y
+
+videoDescriptionShort = Video created at %I:%M %p, %B %e, %Y
+imageDescriptionShort = Image created at %I:%M %p, %B %e, %Y
```

Comment:
Are we going with word created or it should be updated to dated or nothing as recommended by you. See discussion https://github.com/mozilla-b2g/gaia/pull/27049#discussion-diff-22423639R14